src/transports/mod.rs

```diff
@@ -187,15 +188,30 @@ impl<Id: PeerId> InternalTransportType<Id> {
         config: TransportConfig,
         features: PeerNetFeatures,
         local_addr: SocketAddr,
+        total_bytes_received: Arc<RwLock<u64>>,
```

Comment:
For an open source crate like peernet don't you think it would be better to have this in a feature flag ? it's totally optional for the health of peernet. But it implies to have everything under feature flag maybe just open an issue about it for the future.

Reply:
yes you 're right, labnet is used by Leo today so i can work on that this afternoon

Reply:
ok